bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/config/HueBridgeConfig.java

```diff
@@ -42,6 +42,7 @@ public void setIpAddress(String ipAddress) {
     }
 
     public int getPort() {
+        Integer port = this.port;
```

Comment:
Maybe better to use localPort or checkPort or something to avoid "hidding" of class member.

Reply:
Renamed thePort